bgpd/bgp_label.h

```diff
@@ -58,7 +58,7 @@ static inline int bgp_is_withdraw_label(mpls_label_t *label)
 
 	/* The check on pkt[2] for 0x00 or 0x02 is in case bgp_set_valid_label()
 	 * was called on the withdraw label */
-	if ((pkt[0] == 0x80) && (pkt[1] == 0x00)
+	if (((pkt[0] == 0x80) || (pkt[0]) == 0x00) && (pkt[1] == 0x00)
 	    && ((pkt[2] == 0x00) || (pkt[2] == 0x02)))
```

Comment:
The label no 0x000002(which got enabled by this new code), i tried searching if it has to do anything with withdraw label, i couldn't find any info.
On the other hand validation for label no 0x800002(which was already present in the old code), i am not sure why it was added, is it handling any FRR specific case ? if anyone has any info pls let me know ?